manifests/freenas-12-stable.json

```diff
@@ -109,7 +108,6 @@
 				"emulators_mtools_UNSET+=X11",
 				"emulators_openvm-tools_UNSET+=ICU",
 				"editors_jed_UNSET+=X11",
-				"devel_gdb_SET+=PYTHON",
```

Comment:
Keep this, its the default but its safer to keep it here the options we explicitly need in case default ever changes.

Reply:
Same for this, it's a duplicate